tests/dovecot.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# This Modularity Testing Framework helps you to write tests for modules
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Jan Scotka <jscotka@redhat.com>
+#
+
+import socket
+import pexpect
+from moduleframework import module_framework
+
+
+class DovecotSanityTests(module_framework.AvocadoTest):
+    """
+    :avocado: enable
+    """
+
+    def test_connect_to_dovecot_over_openssl(self):
+        self.start()
+        #session = ShellSession("openssl s_client -connect localhost:%s -starttls imap" % self.getConfig()['service']['port'])
+        #session.cmd("a login dummy redhat")
+        #session.cmd("a logout")
+        session = pexpect.spawn("telnet localhost %s" % self.getConfig()['service']['port'])
```

Comment:
I'm not sure if this is okay. if you would like to test openssl, you should use `openssl` command as line #36, this test name `test_connect_to_dovecot_over_openssl` is bad
second thing is that using pexpect should probably also check some output, I'm not sure with return code, in case `telnet localhost %s` return another RC than 0 cause exception, or it also pass, becuase it is based on interaction. and also I expected something like line #45 (some line closing)

Reply:
I have changed it to _telnet which works fine and properly.

I have some troubles with openssl.